web/src/beta/lib/core/StoryPanel/index.tsx

```diff
@@ -114,8 +113,8 @@ export const StoryPanel = memo(
 
 export default StoryPanel;
 
-const PanelWrapper = styled.div`
+const PanelWrapper = styled.div<{ bgColor?: string }>`
   flex: 0 0 ${STORY_PANEL_WIDTH}px;
-  background: #f1f1f1;
+  background: ${({ bgColor }) => (bgColor ? bgColor : "#f1f1f1")};
```

Comment:
Can keep things a little simpler/cleaner like this
```suggestion
  background: ${({ bgColor }) => bgColor || "#f1f1f1"};
```

Reply:
I will remove it all because it is set on convert story